### ISSUE TITLE
No default for /start in telegram

### ISSUE THREAD

**Issue by nasfarley88** [LanceMaverick/skybeard-2#135]
When someone messages a bot for the first time, the telegram client types `/start`. It might be good for skybeard to (by default) show the help message, or even have a custom message set in the config.

**Comment by LanceMaverick** [comment]
I agree. Is there a way to catch these first messages other than just registering the command /start?

**Comment by nasfarley88** [comment]
Don't know, but we could test by running debugbeard in debug mode with a new bot to see what the json of the first message looks like.
